categorize.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+import collections
+import re
+import string
+import sys
+
+# The bins we categorize artifacts as
+Bins = collections.namedtuple(
```

Comment:
can we rename to "executable" or "binaries"? I thought this meant bin as in bin-packing, on the first read.

Reply:
It actually does mean bins as in categories, not binaries. I changed it to 'buckets' which is less ambiguous when we are talking about categorizing into buckets.